### PR TITLE
chore: Introduce fluent -> elasticsearch soak

### LINKED ISSUE #9996: Introduce soak elasticsearch involved

### PR DESCRIPTION
**Pull request by blt**
Resolves #9996

Signed-off-by: Brian L. Troutwine <brian@troutwine.us>

### CODE REVIEW

**Comment by spencergilbert** [comment]
Min value = 0.00 😆

**Comment by blt** [comment]
> Min value = 0.00 laughing - did we crash the soak test again?

I find it doubtful in this case. Consider that the baseline varies so much from the comparison. That said, I don't see the same behavior locally so something wacky is happening.

**Comment by blt** [comment]
Seems more likely -- per 37dcebd -- that we're pushing more CPU on the blackhole than it's allotted. More fuel for the fire of #9946.

File: soaks/tests/fluent_elasticsearch/terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  metadata {
+    name = "soak"
+  }
+}
+
+module "vector" {
+  source       = "../../../common/terraform/modules/vector"
+  type         = var.type
+  vector_image = var.vector_image
+  test_name    = "datadog_agent_remap_datadog_logs"
```
Comment (spencergilbert) [inline soaks/tests/fluent_elasticsearch/terraform/main.tf:41]: This test name seems wrong?

Comment (blt) [reply soaks/tests/fluent_elasticsearch/terraform/main.tf:41]: Oh yeah, it is. I bet that is wrong in a lot of places and we aren't actually using the value, which is why we haven't noticed. Good find.

Comment (spencergilbert) [reply soaks/tests/fluent_elasticsearch/terraform/main.tf:41]: _awkwardly looks at the other PRs I've approved and not noticed it_

Comment (blt) [reply soaks/tests/fluent_elasticsearch/terraform/main.tf:41]: https://github.com/vectordotdev/vector/pull/10003

**Review by tobz — APPROVED** [review]

**Pull request merged**